mirrord-layer/src/config.rs

```diff
@@ -29,6 +29,9 @@ pub struct LayerConfig {
     #[envconfig(from = "MIRRORD_AGENT_TTL", default = "0")]
     pub agent_ttl: u16,
 
+    #[envconfig(from = "MIRRORD_AGENT_TCP_STEAL_TRAFFIC", default = "false")]
+    pub agent_steal_tcp_traffic: bool,
```

Comment:
`agent_tcp_steal_traffic` to be consistent with the envconfig. Or change the envconfig.